scripts/houston.coffee

```diff
@@ -0,0 +1,15 @@
+# Description:
+#   Houston quotes!
+
+answers = [
+  "Let's get f**ked up",
+  "I'm shy",
+  'I feel nothing',
+  'Pick a number',
+  "I don't like being friends with people I work with",
+  "I don't know about that"
+]
+
+module.exports = (robot) ->
+  robot.respond /houston me/i, (msg) ->
+    msg.random answers
```

Comment:
In order for hsbot to send the message this should be `msg.send msg.random answers`